viadot/task_utils.py

```diff
@@ -599,3 +597,76 @@ def adls_bulk_upload(
             sp_credentials_secret=adls_sp_credentials_secret,
             overwrite=adls_overwrite,
         )
+
+
+@task(timeout=3600)
+def anonymize_df(
```

Comment:
Maybe you should change the function name to `anonymize_df_columns` to make it clearer.